apps/web/lib/app/authenticator.tsx

```diff
@@ -55,17 +63,17 @@ export function withAuthentication(
 
 		return (
 			<>
-				{!user || loading ? (
+				{(!user || loading) && showPageSkeleton ? (
 					<TeamPageSkeleton />
 				) : (
 					<>
 						<Component {...props} />
-						<BackdropLoader
+						{/* <BackdropLoader
```

Comment:
@paradoxe35 why this code commented!?

Reply:
We no longer use this loading component, we have replaced it with a skeleton component.
It created an empty DOM element that overly other elements and made them impossible to click.

Reply:
ok, so maybe we can remove that code completely in such case? :)

Reply:
We can remove it